### PR TITLE
Add work section and About Myself

### PR DESCRIPTION
**Pull request by kiboma2021**
Adding the middle section of the mobile website on work and about myself

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,6 +40,129 @@ <h5>LET’S CONNECT</h5>
...
+            Languages
+            <img src="imgs/Icon-Down.png" alt="#" />
+          </li>
+        </ul>
+        <ul class="lang-prog">
+          <li><img src="imgs/JS.png" alt="#" />Javascript</li>
+          <li><img src="imgs/HTML.png" alt="#" />HTML</li>
+          <li><img src="imgs/CSS.png" alt="#" />CSS</li>
+        </ul>
+        <ul class="lang-section hg">
```
Comment (roshan-bajgain) [inline index.html:155]: - [ ] I would suggest you make the spacing between the logo and content to stick with the Figma design and see the content it's different from your color and font.
### your design
figma-about
### Figma design
about-his

File: index.html [hunk 2]
```diff
@@ -40,6 +40,129 @@ <h5>LET’S CONNECT</h5>
...
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="flex ocontent">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="submit">See Project</button>
+        </div>
```
Comment (roshan-bajgain) [inline index.html:85]: - [ ] The button (e.g., "See project") is a link a or button tag, and follow the button styling: Active, Hover, and Pressed (see design). Kindly fix the Hover and pressed for your all buttons.
### Figma design
figma-figma
### When user Hover your buttons
your-hover
### when user pressed your buttons
your-pressed

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @kiboma2021,

 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight :tada: :champagne: 
- Nice PR description and title :+1: 
- Nice README file :heavy_check_mark:
- No linter error :heavy_check_mark:
- GitHub flow is followed :+1:


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -40,6 +40,129 @@ <h5>LET’S CONNECT</h5>
...
```
[the remark this answers is not in the document]
Comment (kiboma2021) [reply index.html:65]: #FFFFFF is flagged by linter 

 15:21  ✖  Expected "#FFFFFF" to be "#ffffff"           color-hex-case                       
  15:21  ✖  Expected "#FFFFFF" to be "#FFF"              color-hex-length

**Pull request merged**